### ISSUE TITLE
Why the validation loss can be less than training loss?

### ISSUE THREAD

**Issue by likecoffee** [keras-team/keras#2330]
I run the keras exmaple of IMDB_LSTM, I get the result like
Epoch 1/15
20000/20000 [==============================] - 42s - loss: 0.5788 - acc: 0.6842 - val_loss: 0.4813 - val_acc: 0.7658

The validation loss and validation accuaracy is better than the training ones.
I wonder is that a right case?
